tests/test_embed/test_interpreter.cpp

```diff
@@ -173,6 +173,18 @@ bool has_pybind11_internals_static() {
 
 TEST_CASE("Restart the interpreter") {
     // Verify pre-restart state.
+    fprintf(stdout,
+            "\nLOOOK PYTHONPATH %s\n",
+            py::module_::import("os")
```

Comment:
Wouldn't it be cleaner to just use the py::print API here?

Reply:
This is/was just for debugging, I'll remove this before taking this PR out of draft mode, after adding a unit test for `PYTHONPATH` processing.

I think `py::print` could do indirections. For debugging I generally try to go straight down to the bare metal.